includes/create-theme/theme-styles.php

```diff
@@ -26,15 +26,18 @@ public static function update_style_css( $style_css, $theme ) {
 		$author_uri    = $theme['author_uri'];
 		$wp_version    = CBT_Theme_Utils::get_current_wordpress_version();
 		$wp_min        = $current_theme->get( 'RequiresWP' );
-		$version       = $theme['version'];
-		$requires_php  = $current_theme->get( 'RequiresPHP' );
-		$text_domain   = $theme['slug'];
-		$template      = $current_theme->get( 'Template' ) ? "\n" . 'Template: ' . $current_theme->get( 'Template' ) : '';
-		$license       = $style_data['License'] ? $style_data['License'] : 'GNU General Public License v2 or later';
-		$license_uri   = $style_data['LicenseURI'] ? $style_data['LicenseURI'] : 'http://www.gnu.org/licenses/gpl-2.0.html';
-		$tags          = CBT_Theme_Tags::theme_tags_list( $theme );
-		$css_contents  = $css_contents ? "\n\n" . $css_contents : '';
-		$copyright     = '';
+		if ( ! $wp_min ) {
+			$wp_min = '5.9';
```

Comment:
It looks like we've hard-coded `6.0` elsewhere, maybe we should go with that rather than `5.9`?

Reply:
I updated to `6.0`, but it made me wonder why we're going with `6.0` vs `5.9` since that's when block themes were introduced.

Reply:
I wasn't sure either - looking at when we added `6.0` to the readme class, I found this [related issue](https://github.com/WordPress/create-block-theme/issues/355) where it looks like we decided based on this:

> The requirement to include the index.php file was removed in WordPress 6.0, so I recommend increasing it to at least that version.

In that case, I think we're fine to stick with `6.0`.